src/DynamoCore/Properties/Resources.en-US.resx

```diff
@@ -653,4 +653,13 @@ value : bool = false</value>
     <value>Home</value>
     <comment>The default name of home workspace</comment>
   </data>
+  <data name="UnresolvedNodesWarningShortMessage" xml:space="preserve">
+    <value>This graph cannot be saved until unresolved nodes are removed or resolved.</value>
```

Comment:
My understanding was that the original graph file cannot be overwritten but can be saved "as". If this is true you may want to reword this text a little bit.